### DISCUSSION TITLE
Acoustic model

### DISCUSSION THREAD

**Discussion by botankit** [coqui-ai/STT#2226] [General]
Is Latest Acoustic model available on coqui zoo is model.tflite (45.1 MB)?
If Yes, but in MODEL_CARD file, the size of  Acoustic model is given as 181 MB. So, From Where I can download that model(181 MB)?
Is English checkpoints file is applicable for all english models?

**Comment by nTeligence** [comment]
Same basic question, the large English language model download page (https://coqui.ai/english/coqui/v1.0.0-large-vocab#download) indicates that the model.tflite should be 181 MB, but the file that actually downloads is only ~47MB. Has Coqui as a company discontinued the distribution of larger (hopefully more accurate) open source models under the Apache 2.0 license, so that they can commercialize them? If not, can someone at Coqui update this link, so that the 181MB large model can be downloaded. Thanks in advance.

**Comment by reuben** [comment]
The ~180MB acoustic model is the same as the 47MB one, but without quantization. There is a very small difference in accuracy but a very large difference in performance, so we don't ship the unquantized model. You can export it yourself from the checkpoint, by disabling quantization.
